packages/bpk-component-horizontal-nav/src/BpkHorizontalNav.module.scss

```diff
@@ -20,6 +20,8 @@
 @use '../../unstable__bpk-mixins/borders';
 
 .bpk-horizontal-nav {
+  position: relative;
```

Comment:
Nice, was just about to raise this: https://github.com/Skyscanner/backpack/pull/3336